dev/run-tests.py

```diff
@@ -294,7 +294,8 @@ def build_spark_sbt(hadoop_version):
     sbt_goals = ["package",
                  "assembly/assembly",
                  "streaming-kafka-assembly/assembly",
-                 "streaming-flume-assembly/assembly"]
+                 "streaming-flume-assembly/assembly",
+                 "streaming-mqtt-assembly/assembly"]
```

Comment:
@prabeesh you can add a sbt goal `streaming-mqtt-assembly/test:package` here. It should generate the test jar.